README.md

```diff
@@ -85,35 +85,48 @@ Comments:
 - 'y = y[t-1] * rog' is formula (equation)
 
 Programming entry point
------------------------
+=======================
 ```python
-from xl import XlSheet
+from xlsmodel.xl import XlSheet
 XlSheet(filename="test1.xls", sheet="input_sheet_v1", anchor="A1").save(sheet="output_v2")
 ```
 
-Command line call 
------------------
-```
-python xl.py <filname> <sheet> <anchor>
-   <filename>: filename ot path to .xls file
-   <sheet>: string representing sheet name or integer based at 1, defaults to first sheet in file  
-   <anchor>: string with A1 style reference pointing to start of datablock on sheet, defaults to "A1"
-
-python xl.py test0.xls
-python xl.py test0.xls 1 A1
-python xl.py test0.xls sheet1 A1
+Command line interface
+======================
+```bash
+# TODO(dmu) LOW: Is this usage really needed here? We can get it any time with:
+# python -m xlsmodel.xl --help
+usage: python -m xlsmodel.xl [-h] filename [sheet] [anchor]
```

Comment:
При необходимости можно использовать docopt, там более гибкие возможности
настройки интерфейса командной строки, сейчас остановимся на argparse.
8 июня 2016 г. 12:40 пользователь "Dmitry Mugtasimov" <
notifications@github.com> написал:

> In README.md
> <https://github.com/epogrebnyak/make-xls-model-2/pull/9#discussion_r66224444>
> :
>
> > ------------------
> > -```
> > -python xl.py <filname> <sheet> <anchor>
> > -   <filename>: filename ot path to .xls file
> > -   <sheet>: string representing sheet name or integer based at 1, defaults to first sheet in file
> > -   <anchor>: string with A1 style reference pointing to start of datablock on sheet, defaults to "A1"
> > -
> > -python xl.py test0.xls
> > -python xl.py test0.xls 1 A1
> > -python xl.py test0.xls sheet1 A1
> > +Command line interface
> > +======================
> > +```bash
> > +# TODO(dmu) LOW: Is this usage really needed here? We can get it any time with:
> > +# python -m xlsmodel.xl --help
> > +usage: python -m xlsmodel.xl [-h] filename [sheet] [anchor]
>
> 1) да, лучше переименовать, но пока я оставил оригинальное название
> 2) это автоматически генерируется пакетом argparse по правилам: сначала
> идут опции (которые начинаются с минуса "-", а потом аргументы, в т.ч.
> опциональные, по-хорошему опциональные аргументы - это не очень правильно -
> они должны стать опциями, но я пока сохранил оригинальный синтаксис)
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/epogrebnyak/make-xls-model-2/pull/9/files/a34d28364be609249eb2f8a90c6b1c2061cb57fc..7eb19ce58153ef0fe04d18bf044187aaec1a2715#r66224444>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AI1grtM3sOYcUC7TvWjkWWFcZzfQBqmjks5qJo4EgaJpZM4IwUM5>
> .
>